widgets/mpd_all.lua

```diff
@@ -9,6 +9,7 @@ local io = { popen = io.popen }
 local setmetatable = setmetatable
 local string = { gmatch = string.gmatch }
 local helpers = require("vicious.helpers")
+local math = require("math")
```

Comment:
Since only `math.floor` is used, I think `local math = { floor = math.floor }` would be more implicit.